docs/howtos/locally-published-spm-in-firefox.md

```diff
@@ -0,0 +1,105 @@
+1. Know what version of `rust-components-swift` was used beforehand. You can find this by checking the git diff on `firefox-ios/Client.xcodeproj/project.pbxproj`.
+1. Run:
+    ```bash
+    $ ./rust_components_local.sh --disable <VERSION> ../rust-components-swift
+    ```
+1. Then, make sure to reset packages cache in Xcode. This forces Xcode to remove any previously cached versions of the Rust components.
+    - You can reset package caches by going to `File -> Packages -> Reset Package Caches`
+
+
+> If you happen to change branches in `rust-components-swift`, you will need to disable then re-enable local development. The script is not currently smart enough to switch branches. Alternatively, keep the branch in `rust-components-swift` the same. `rust-components-swift` serves only as a release surface so there is little use to switching branches and pushing changes to it, unless you are changing something related to the release process.
+
+## Using the manual flow
+It's important to note the automated flow runs through those steps in a script, so if possible use the script as it's a tedious manual process. However, if the script is failing or you would like to run the manual process for any other reason follow the following steps.
+
+### Building the xcframework
+To build the [xcframework](https://developer.apple.com/documentation/swift_packages/distributing_binary_frameworks_as_swift_packages) do the following:
+1. In your local checkout of `application-services`, navigate to [`megazords/ios-rust/`](https://github.com/mozilla/application-services/tree/main/megazords/ios-rust)
+1. Run the `build-xcframework.sh` script: 
+```bash
+$ ./build-xcframework.sh
+```
+This will produce a file name `MozillaRustComponents.xcframework.zip` that contains the following, built for all our target iOS platforms.
+- The compiled Rust code for all the crates listed in `Cargo.toml` as a static library
+- The C header files and [Swift module maps](https://clang.llvm.org/docs/Modules.html) for the components
+
+## Include the xcframework in a local checkout of `rust-components-swift`
+After you generated the `MozillaRustComponents.xcframework.zip` in the previous step, do the following to include it in a local checkout of `rust-components-swift`:
+1. Unzip the `MozillaRustComponents.xcframework.zip` into the `rust-components-swift` repository: (Assuming you are in the root of the `rust-components-swift` directory and `application-services` is a neighbor directory)
+    ```sh
+     unzip  ../application-services/megazords/ios-rust/MozillaRustComponents.xcframework.zip -d .
+    ```
+1. Change the `Package.swift`'s reference to the xcframework to point to unzipped folder. You can do this by uncommenting the following line:
+    ```swift
+        path: "./MozillaRustComponents.xcframework"
+    ```
+    and commenting out the following lines:
+    ```swift
+        url: url,
+        checksum: checksum,
+    ```
```

Comment:
I'm seeing this, which (a) seems wrong compared to the instructions, but (b) seems to be where we want to end up? 
![image](https://user-images.githubusercontent.com/83924/168971332-cbbd7c7f-1386-4893-a332-db6526c0c0ad.png)

Reply:
Yeah, what you see in the result of the script, which already does the work of replacing the lines needed

The manual steps assume you're starting from scratch without running the script